src/formatDate.js

```diff
@@ -8,7 +8,85 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  // find out which separator
+  // to split the string
+  // to convert to an array of numbers
+  const pastSeparator = fromFormat[3];
+  const oldFormat = date.split(pastSeparator);
```

Comment:
The variable name 'pastSeparator' might be a bit misleading, as it suggests a historical separator. A name like 'currentSeparator' or 'fromSeparator' would be more accurate and consistent with the context.